### DISCUSSION TITLE
shopping for hardware 
Is there a better list of hardware with links supplier links?

### DISCUSSION THREAD

**Discussion by stovedoctor** [HASwitchPlate/openHASP#452] [Hardware]
Well thanks to Drzzs I am now sucked into openhasp.
I set up a landbond and the uses are endless.
Looking for less expensive options 
for larger display for just wallnad desk panel no need for relays.
Would like to be able to connect  PIR and a TH sensors. 

Is there a better list of hardware with links supplier links?

I don't mind waiting but I have been burned 4 times with AliExpress.

https://a.aliexpress.com/_m0ss0Tc

Amazon us https://a.co/d/hA0GLja

WT32-SC01 PLUS With 3.5 inch 320X480
![Screenshot_20230325-104524.png](https://user-images.githubusercontent.com/60519485/227733238-873dd036-0f49-44d4-b19d-d71123554aca.png)

**Comment by stovedoctor** [comment]
I'm new to this so maybe users posting list of their hardware with - 
- pros and cons 
- supplier price
- stl files for 3d Inclosure
- power supply options
- link to their page and yaml examples 

I'm not sure where to start when the sourcing display and esp boards separately.

**Comment by adorobis** [comment]
> WT32-SC01 PLUS With 3.5 inch 320X480

Watch out, the one you have on your screenshot is just a debugging tool. I have bought recently this one, delivery was in around 2 weeks in Europe.
https://pl.aliexpress.com/item/1005004898785933.html
Just make sure to select the Plus version.

**Reply by stovedoctor** [reply]
Thanks 
Hoping others chime in with hardware options. The screenshot just one I saw others talk about.

**Comment by stovedoctor** [comment]
I just found this for US orders 
https://www.digikey.com/en/products/detail/seeed-technology-co-ltd/104990762/17180805?s=N4IgTCBcDaIOoBUDMYC0BlAwgBgIwAIAFAGQFV19V8BRdQlEAXQF8g

28.62$ with 4 day FedX shipping  only 6.99$ shipping
Same shipping price for multiple units.
This is the manufacturer page of same product part number. 
Is this the same thing?
https://www.seeedstudio.com/WT32-3-5-Inch-Display-p-5542.html?queryID=0e11311575c8304c5483beb4da560b8c&objectID=5542&indexName=bazaar_retailer_products

**Reply by stovedoctor** [reply]
I received 2  WT32-SC01 PLUS units  yesterday in perfect shape. From digikey 
$68.67 with shipping and tax.

I had a hard time finding documentation in ONE PLACE how to make it connect for web install without using my FTDI adapter and go into boot mode.. 
What version on openhasp that works with it. Many trial and errors. 
Turned out I had to ground gpioo and ground to get into boot mode.
And use beta version 7 of openhasp for it to work.
Then I had to upgrade my other device to the same version for it to work in home assistant. 
And remove and upgrade home assistant integration from version 6 to the beta version 7 in hacs. 
Took some time but it works 
I like the new file editor in version 7.

**Comment by htvekov** [comment]
Well, the 'official' openHASP recommandations for new project devices are:
ESP32-S3, min. 8 MB flash, 2 MB PSram and a capacitive display.

Both the [WT32-SC01 Plus](https://www.openhasp.com/0.7.0/hardware/wireless-tag/wt32-sc01-plus/) and the three 480x800 res. [Sunton variants](https://www.openhasp.com/0.7.0/hardware/sunton/esp32-8048s0xx/) are all great candidates for desktop devices.

Enclosure designs exists AFAIK for most (all) of these devices.
Also numerous page examples shared through Discord, in openHASP documentation and various github repos.

I personally have two variants of the Sunton dev boards and the WT32-SC01 Plus.
The 7" version of the Sunton has a 'standard' TFT display, where the 5" version is fitted with a higher quality IPS display (much better viewing angles). Both are equally priced and I got mine directly from the [Sunton Store](https://www.aliexpress.com/item/1005004952726089.html?spm=a2g0o.productlist.main.1.31497c11vAtTST&algo_pvid=68e20324-d890-4f11-9b99-869b1fa8b897&aem_p4p_detail=202304030224055260686470645040020991393&algo_exp_id=68e20324-d890-4f11-9b99-869b1fa8b897-0&pdp_ext_f=%7B%22sku_id%22%3A%2212000031135431789%22%7D&pdp_npi=3%40dis%21USD%2136.11%2136.11%21%21%21%21%21%4021224cdb16805138458221339d070a%2112000031135431789%21sea%21DK%211815108366&curPageLogUid=NbJmwzoA37qZ&ad_pvid=202304030224055260686470645040020991393_1&ad_pvid=202304030224055260686470645040020991393_1) on AliExpress.
